src/app/website/shared/contentful/contentful.service.ts

```diff
@@ -0,0 +1,115 @@
+import { Http, Response } from '@angular/http';
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/toPromise';
+@Injectable()
+export class ContentfulService {
+
+  constructor(private http: Http) { }
+
+  public getPossibilitiesData(): any {
+    return this.http.get('https://cdn.contentful.com/spaces/l99z1yyviltd/entries?access_token=5f91dcaac99947796b2f772dba878c5b58fab81d02375c7b9d3f26a9e3a0f45b&content_type=productCoPossibilities')
```

Comment:
duplication. You have to think about re-usability here.  If tomorrow the api end point changes you will have to change this form line no 11, 23,33 etc.. This is not scalable and not re-usable.